app.go

```diff
@@ -264,3 +264,58 @@ func (c *Client) RevokeTokens(ctx context.Context, before *time.Time) (*Response
 	err := c.makeRequest(ctx, http.MethodPatch, "app", nil, setting, &resp)
 	return &resp, err
 }
+
+type PushProviderType string
+
+const (
+	PushProviderTypeFirebase PushProviderType = "firebase"
+	PushProviderTypeAPN      PushProviderType = "apn"
+	PushProviderTypeHuawei   PushProviderType = "huawei"
+	PushProviderTypeXiaomi   PushProviderType = "xiaomi"
```

Comment:
could you remove type probably?

Reply:
just realized this exists already in device.go ✅